### DISCUSSION TITLE
constructing CFAD

### DISCUSSION THREAD

**Discussion by MohitDahliya** [ARM-DOE/pyart#1459] [Q&A]
How to construct a contoured frequency by altitude diagram (CFAD) plot

**Comment by zssherman** [comment]
@MohitDahliya [RadTraq](https://github.com/ARM-Development/RadTraQ) has some functionality for this. If your using xradar and Py-ART, you can provide a xarray dataset to that. Another option is in the py-art google post here:
https://groups.google.com/g/pyart-users/c/cpObew7XpwI/m/QEkoVbGkDgAJ

We do not however use the py-art google group anymore so the code in that post you would have to reach out to that author if you have questions and issues etc.

Hope that helps!
